.github/workflows/build.yaml

```diff
@@ -0,0 +1,14 @@
+name: "Build"
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: generate-html
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+      - name: build
+        run: ./repo.sh docs
```

Comment:
Maybe we can improve this job to store generated docs as a build artifact?

Reply:
Sure, any suggestion?
Should it be in same PR or follow up?

Reply:
here is the example how to achieve this with gh actions https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts#example.
Not a blocker for to this PR

Reply:
Let's handle it as part of #5